packages/@okta/vuepress-site/docs/reference/api/features/index.md

```diff
@@ -619,7 +619,7 @@ The Stage object has following properties:
 If a Feature's stage `value` is `EA`, the `status` is `null` and not returned. If the value is `BETA` the status is `OPEN` or `CLOSED` depending on the `BETA` feature
 being manageable or not.
 
-> Note: If a Feature's stage is Open Beta, it can be updated in Preview cells only. If a Feature's stage is Closed Beta, it can only be disabled in Preview cells.
+> **Note:** If a Feature's stage is Open Beta, it can be updated in Preview cells only. If a Feature's stage is Closed Beta, it can only be disabled in Preview cells.
```

Comment:
weird that this one doesn't show up as bold like the others?

Reply:
Only in GitHub. It does show up bold in VuePress.